### PR TITLE
Support filtering for comment-based unit tests

### LINKED ISSUE #3162: Support filtering for comment-based unit tests
The [rule-test-launcher](https://github.com/SonarSource/SonarJS/blob/master/eslint-bridge/tests/rules/rule-test-launcher.test.ts) does not support Jest's test filtering. It should, as when we develop a rule, it is convenient to run the tests for your rule only, for readability and performance reasons.

### PR DESCRIPTION
**Pull request by ilia-kebets-sonarsource**
Fixes #3162

I have added a new script `ntest` (as "new test") as adding the filtering to `npm test` such as `npm test -t=your-rule` takes around 15s to run since it is going through all files defined by `<rootDir>/tests/**/*.test.t`.

### CODE REVIEW

File: eslint-bridge/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+## Test
+
+The ESLint bridge has unit tests that use [ESLint's RuleTester API](https://eslint.org/docs/developer-guide/nodejs-api#ruletester).
+
+### Comment-based tests
+
+We implemented a new format to write unit tests for ESLint using a more readable format.
+
+- You can launch them using: `npm run ntest`
```
Comment (vilchik-elena) [inline eslint-bridge/README.md:12]: I don't think we should document this one. Why someone would want to run all comment-based tests?

File: eslint-bridge/package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
     "clear": "tsc -b src tests --clean",
     "check-format": "prettier --list-different \"{src,tests}/**/!(*.lint).ts\"",
     "test": "jest",
+    "ntest": "npm test tests/rules/rule-test-launcher.test.ts",
```
Comment (vilchik-elena) [inline eslint-bridge/package.json:10]: I don't get reasoning behind this name `ntest`. It stands for `new test` but what is new about it? How about `rule-test`?

**Review by vilchik-elena — CHANGES_REQUESTED** [review]
It does not seem to work for me, all rules are executed.

File: eslint-bridge/package.json (hunk 2 above)
Comment (ilia-kebets-sonarsource) [reply eslint-bridge/package.json]: Yeah, I'm not so confident with this alias `ntest`. 
It filters out the comments-based tests from all unit tests. I'll ask Victor if he feels creative.

Comment (ilia-kebets-sonarsource) [reply eslint-bridge/package.json]: I propose `ctest` for "comment-based tests"

**Review by vilchik-elena — CHANGES_REQUESTED** [review]

**Review by vilchik-elena — APPROVED** [review]

**Pull request merged**